src/DSharp.Mscorlib/NonStandard/Collections/Generic/List.cs

```diff
@@ -7,9 +8,9 @@ public sealed partial class List<T>
         [Obsolete(ObsoleteConsts.MESSAGE_ON_OBSOLETE, ObsoleteConsts.ERROR_ON_OBSOLETE)]
         public extern void ForEach(ListCallback<T> callback);
 
-        [ScriptName("push")]
+        [DSharpScriptMemberName("addRangeParams")]
         [Obsolete(ObsoleteConsts.MESSAGE_ON_OBSOLETE, ObsoleteConsts.ERROR_ON_OBSOLETE)]
-        public extern void AddRange(params T[] items);
+        public extern void AddRangeParams(params T[] items);
```

Comment:
Can we keep this as `AddRange` as the public interface, as we already have a lot of usages for this method which we should migrate away from instead of having to change them all when we upgrade

Reply:
the issue with is that ScriptAlias doesn't work for overloads (it always picks the first one instead) so we need to drop the CLR implementation if we want to maintain this one